frontend/packages/kubevirt-plugin/src/utils/validations/vm/disk.ts

```diff
@@ -138,8 +139,7 @@ export const validateDisk = (
       validations.pvc = validatePVCName(pvcName, usedPVCNames);
     }
 
-    // TODO: implement CDROM disk bus validation
-    if (disk.getType() === DiskType.DISK) {
+    if (diskType === DiskType.DISK || diskType === DiskType.CDROM) {
```

Comment:
this is useful for LUN as well. Can we do this?
```suggestion
    if (disk.getType() !== DiskType.FLOPPY) {
```